src/site/content/en/handbook/web-dev-components/index.md

```diff
@@ -149,11 +165,37 @@ Use the codelab aside to link to an associated codelab.
 
 ## Banner
 
-{% Banner %}This is an info banner with a <a href="#">link</a>.{% endBanner %}
+```text
+{% raw %}{% verbatim %}
+{% Banner %}This is an info banner with a [link](#).{% endBanner %}
```

Comment:
I think the raw tag is making the markdown link show up. Might be good to just change the text to say "This is an info banner. It supports markdown."

Reply:
screenshot:

![image](https://user-images.githubusercontent.com/1066253/66614625-9afa1c80-eb7e-11e9-81ac-78a66d9e9fd5.png)

Reply:
Changed